velox/functions/prestosql/tests/ComparisonsTest.cpp

```diff
@@ -111,6 +111,26 @@ TEST_F(ComparisonsTest, betweenDate) {
   }
 }
 
+TEST_F(ComparisonsTest, betweenTimestamp) {
+  using util::fromTimestampString;
+
+  std::vector<std::tuple<Timestamp, bool>> testTimestamp = {
```

Comment:
Perhaps, the test will be easier to read if written this way.

```
auto between = [] (const std::string& timestamp) {
   return evaluateOne<bool>("c0 between ....", fromTimestampString(timestamp));
}

EXPECT_FALSE(between("2019-02-28 10:00:00.499"));
EXPECT_TRUE(between("2019-02-28 10:00:00.500"));
...
```

Reply:
Refactored test case with advice.